extensions/objects/templates/ListOfSetsOfHtmlStringsEditor.js

```diff
@@ -0,0 +1,103 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+oppia.directive('listOfSetsOfHtmlStringsEditor', [
+  'OBJECT_EDITOR_URL_PREFIX', 'UrlInterpolationService',
+  function(OBJECT_EDITOR_URL_PREFIX, UrlInterpolationService) {
+    return {
+      restrict: 'E',
+      scope: {
+        getInitArgs: '&',
+        value: '='
+      },
+      templateUrl: UrlInterpolationService.getExtensionResourceUrl(
+        '/objects/templates/' +
+        'list_of_sets_of_html_strings_editor_directive.html'),
+      controller: ['$scope', function($scope) {
+        var errorMessage = '';
+        $scope.selectedRank = '';
+
+        if (!$scope.value) {
+          $scope.value = [];
+        }
+        if (!$scope.maxPrevIndex) {
```

Comment:
This does not appear to be previously defined, so you can probably just assign it 0.

Reply:
Done.

Reply:
...and remove this block :)

Reply:
Removed, done!